power/ReadVcdActivities.cc

```diff
@@ -196,6 +225,13 @@ ReadVcdActivities::setVarActivity(const char *pin_name,
       checkClkPeriod(pin, transition_count);
     power_->setUserActivity(pin, activity, duty, PwrActivityOrigin::vcd);
     annotated_pins_.insert(pin);
+    //const char *pname = cmd_network_->pathName(pin);
+    //std::cout << "VCD Annotated pin: " << pin_name  << " port: " << pname << " tr count: " << transition_count
+    //          << " activity: " << activity << " duty: " << duty << std::endl;
+  }
+
+  if (pins.empty()) {
+    std::cout << "WARNING: Missed annotating pin: " << pin_name << std::endl;
```

Comment:
Use native warning

Reply:
Adding a TODO to remove this code altogether, this is internal debugging while in dev mode.